persistent-test/persistent-test.cabal

```diff
@@ -187,6 +187,8 @@ executable persistent-test
                  , resourcet
                  , scientific
 
+  if !flag(postgresql) && !flag(mysql) && !flag(mongodb) && !flag(zookeeper)
+      cpp-options: -DWITH_SQLITE -DDEBUG
```

Comment:
This seems a little weird—why not have a flag for SQLite?

Reply:
It's just doing for test/main.hs what was already being done for the main test modules in the library

Reply:
Oh right, it's this way so that SQLite is the default. 👍